examples/Test_domain/connectivity/2000/example_conn.csv

```diff
@@ -0,0 +1,14 @@
+# Transfer probability matrix for tests only,,,,,,,,,,
+# Row names are the SOURCE sites. Column names indicate the RECEIVING reef.,,,,,,,,,,
+# Refer to the associated cross-reference table for reef locations and reef ids.,,,,,,,,,,
```

Comment:
Please don't edit these with Excel, it automatically messes up the formatting.

Delete the additional columns at end of line.